mx.py

```diff
@@ -4243,7 +4247,8 @@ def _versionGetter(suite):
 
     version = _versionGetter(s)
     log('Deploying {0} distributions for version {1}'.format(s.name, version))
-    _deploy_binary_maven(s, _map_to_maven_dist_name(mxMetaName), _mavenGroupId(s), mxMetaJar, version, repo.name, repo.url, settingsXml=args.settings, dryRun=args.dry_run)
+    if not args.platform_dependent:
+        _deploy_binary_maven(s, _map_to_maven_dist_name(mxMetaName), _mavenGroupId(s), mxMetaJar, version, repo.name, repo.url, settingsXml=args.settings, dryRun=args.dry_run)
```

Comment:
That line is deploying the "jvmci-mx.jar", which is always platform independent.